notSolid/Test.java

```diff
@@ -0,0 +1,108 @@
+package com.notSolid;
```

Comment:
The package name should be in lower case.
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html